packages/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redocly/openapi-cli",
-  "version": "1.0.0-beta.90",
+  "version": "1.0.0-beta.91",
```

Comment:
Please bump the `openapi-core` version in dependencies as well.

Reply:
Done